sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md

```diff
@@ -1,19 +1,22 @@
 # Release History
 
-## 5.12.0-beta.1 (Unreleased)
+## 5.11.1 (2022-03-17)
 
 ### Features Added
 - Updated the receiver to retry to obtain a new connection if the RequestResponseChannel in the current connection is disposed.
```

Comment:
Should this behavioral update warrant a minor bump @srnagar?

Reply:
@vcolin7 this doesn't require a minor version bump as this behavior change improves reliability and isn't a functional change for the customer.